source/user.scripts/usr/local/emhttp/plugins/user.scripts/Userscripts.page

```diff
@@ -342,15 +350,31 @@ function editDesc(myID) {
   $("#"+origID).tooltipster("enable");
 }
 
+function getModeForShebang(data)
```

Comment:
This is a function of my own creation to determine the type of code in the script from the shebang. In compose.manager i only use yaml and text, while dynamix file manager uses a builtin ace function which is based on the filetype.